Core/Configuration/ServiceConfiguration.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Json;
+
+namespace AeroGear.Mobile.Core.Configuration
+{
+    /// <summary>
+    /// This represents a parsed singleThreadService configuration from JSON configuration.
+    /// </summary>
+    public class ServiceConfiguration : IReadOnlyDictionary<string,string>
```

Comment:
can we add some API docs for the public fields and methods.

Reply:
ok